crates/solidity/outputs/typescript/lib/.github/renovate.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I suggest removing that file, since it can trigger unnecessary GitHub PRs moving forward. We will be tackling this in #25

Reply:
done